src/components/SchedulePage/Schedule.js

```diff
@@ -35,7 +35,7 @@ const SchedulePage = () => {
 	const classes = useStyles();
 	return <Container maxWidth='md'>
 		<Typography align='left' component='h1' variant='h4' className={classes.title}>
-			Schedule Coming Soon!
+			Events Schedule
```

Comment:
Can you change line "Upcoming Workshops" on line 87 of this file to say "Upcoming Events"?